images/kubeflow-pipelines/configs/template.metadata-envoy.apko.yaml

```diff
@@ -0,0 +1,19 @@
+contents:
+  packages:
+    - envoy
+    - kubeflow-pipelines-metadata-envoy-config
+    - gettext
+    - openssl
```

Comment:
nit: can we add these as dependencies to the package instead?

Reply:
self resolving after some side discussion. since the package (`kubeflow-pipelines-metadata-envoy-config`) is a config only package, we want it to remain compatible across various versions